src/Files.App/Views/Layouts/DetailsLayoutPage.xaml.cs

```diff
@@ -579,7 +579,6 @@ private void UpdateColumnLayout()
 
 		private void RootGrid_SizeChanged(object? sender, SizeChangedEventArgs? e)
 		{
-			ColumnsViewModel.SetDesiredSize(Math.Max(0, RootGrid.ActualWidth - 80));
```

Comment:
I think this is needed for when the window is resized

Reply:
This method does nothing

```cs
		public void SetDesiredSize(double width)
		{
			if (TotalWidth > width || TotalWidth < width)
			{
				var proportion = width / TotalWidth;

				//SetColumnSizeProportionally(proportion);
			}
		}
```

Reply:
Can you try to find out when this was commented out? It'll be good to know why we originally had this in place.

Reply:
LOL. That was you.
https://github.com/files-community/Files/commit/976484fbe267f64025287f7602916f477867fd05

Reply:
Good fine! Looks like I left it in so we can add an option to auto size columns to fit.

Reply:
I'm not sure if we should leave as is because I'm going to introduce

Reply:
If we will remove it and find out we need it, I will back it back soon in another PR that will be open as soon as this PR merged.